### PR TITLE
Feature/transects quadrats as samples

### PR DESCRIPTION
**Pull request by dr-shorthair**
As discussed in today's meeting

### CODE REVIEW

**Comment by edmondchuc** [comment]
@dr-shorthair thanks for the PR! 

I am wondering if `tern:sampleType` is required since we have `tern:featureType` on the `tern:FeatureOfInterest` class already. For `tern:Site`, we previously had a `tern:siteType`. But since Sites are also Samples (and Samples are FeatureOfInterest), we can also use `tern:featureType` here too. This extends to other types of Samples as well, e.g. MaterialSample.

File: docs/tern.ttl [hunk 1]
```diff
@@ -2505,6 +2512,12 @@ tern:sampleStorageLocation
   rdfs:subPropertyOf geosparql:hasGeometry ;
   skos:definition "A property that links a [PhysicalSpecimen](#PhysicalSpecimen) to the location [Point](https://w3id.org/tern/ontologies/loc/Point) of where it is stored." ;
 .
+tern:sampleType
+  a rdf:Property ;
+  rdfs:domain tern:Site ;
+  rdfs:label "sample or site type" ;
+  skos:definition "The sample or site type. The value is expected to be from a controlled vocabulary." ;
+.
```
Comment (edmondchuc) [inline docs/tern.ttl:2520]: I'm not sure if it should be `tern:sampleType rdfs:domain tern:Site` here.

**Comment by dr-shorthair** [comment]
I agree. `tern:FeatureType` is enough. I've updated the PR to remove `tern:sampleType`

**Comment by edmondchuc** [comment]
The changes in this PR have been merged into https://github.com/ternaustralia/ontology_tern/pull/23. 

It was easier this way to fix the merge conflicts locally before creating another PR. 

Please see https://github.com/ternaustralia/ontology_tern/pull/23 and review the PR into the `develop` branch.

**Pull request closed without merging**